v2/concordium/types.proto

```diff
@@ -2159,3 +2159,41 @@ message ElectionInfo {
   // List of the currently eligible bakers.
   repeated Baker baker_election_info = 3;
 }
+
+// todo: this is obsolete after a merge with 
+// network dump pr.
+message BooleanResponse {
+  bool value = 1;
+}
+
+// A banned peer
+message BannedPeer {
+    // The IP address of the banned peer.
+    IpAddress ip_address = 1;
```

Comment:
I see you have the IpAddress already defined in this PR. So you can make use in the `PeersInfo`.